### PR TITLE
Add set-namespace blueprint

### PR DESCRIPTION
**Pull request by prachirp**

### CODE REVIEW

File: examples/set-namespace/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/set-namespace .
+kpt fn run set-namespace/configs --fn-path set-namespace/functions
+```
+
+## Expected result
+
+Check all resources have `metadata.namespace` set to `example-ns`:
```
[the remark this answers is not in the document]
Comment (prachirp) [reply examples/set-namespace/README.md:19]: Documented limitation

**Comment by prachirp** [comment]
cc @frankfarzan PTAL

File: examples/set-namespace/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+cluster-scoped]. This example invokes the set-namespace function using
+declarative configuration.
+
+## Function invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/set-namespace .
+kpt fn run set-namespace/configs --fn-path set-namespace/functions
```
Comment (frankfarzan) [inline examples/set-namespace/README.md:14]: why do you need `fn-path`?

Comment (prachirp) [reply examples/set-namespace/README.md:14]: I think it's cleaner not to set the namespace on the function config, only on the configs in the package dir set-namespace/configs. So I put the function config in a separate fn-path set-namespace/functions.

Comment (frankfarzan) [reply examples/set-namespace/README.md:13]: Underlying issue is that function configs should not be processed by functions. That seems like a bug and may be related to https://github.com/GoogleContainerTools/kpt/issues/630

Comment (prachirp) [reply examples/set-namespace/README.md:13]: Same as for annotate-config example. Let's merge these now and update them when there's consensus.

**Comment by prachirp** [comment]
@frankfarzan PTAL

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**